packages/mds-policy-author/api.ts

```diff
@@ -242,7 +242,7 @@ function api(app: express.Express): express.Express {
   /* eslint-disable-next-line @typescript-eslint/no-unused-vars */
   app.use(async (error: Error, req: ApiRequest, res: ApiResponse) => {
     await logger.error(req.method, req.originalUrl, error)
-    return res.status(500).send(error)
+    return res.status(500).send({ error })
```

Comment:
You didn't want to do the fancy stuff with `next`?

Reply:
That's precisely what's happening here, this _is_ the `next` function